### PR TITLE
Update versions to 2.7.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by HelloHorizon**
### What changes are proposed in this pull request?

Update versions to 2.7.0-SNAPSHOT

### Does this PR introduce any user facing changes?
No.

### CODE REVIEW

**Comment by HelloHorizon** [comment]
[79da5ed](https://github.com/Alluxio/alluxio/pull/13656/commits/79da5ed74004591b00ceacf7a49412ce3123f525) fix regression from #13026

File: docs/_config.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ kramdown:
...
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-2.7.0-SNAPSHOT-client.jar
 # For release branches, this should be the Alluxio version in the form of <major>.<minor>, e.g. both 1.5.0 and 1.5.0-RC1 should use "1.5"
 # For master branch, this should be "edge"
 # This should be used to reference versioned pages that are not relative to the docs/ directory
-# e.g. Javadoc:      https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION: 2.6
-# e.g. REST API doc: https://docs.alluxio.io/os/restdoc/{{site.ALLUXIO_MAJOR_VERSION: 2.6
-ALLUXIO_MAJOR_VERSION: 2.6
+# e.g. Javadoc:      https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/master/index.html
+# e.g. REST API doc: https://docs.alluxio.io/os/restdoc/{{site.ALLUXIO_MAJOR_VERSION}}/master/index.html
+ALLUXIO_MAJOR_VERSION: edge
```
Comment (Xenorith) [inline docs/_config.yml:19]: i think this needs to be 2.7 but not 100% certain. i would grep for `ALLUXIO_MAJOR_VERSION` in the docs folder to see how it's used to confirm

Comment (HelloHorizon) [reply docs/_config.yml:19]: I refer to this PR https://github.com/Alluxio/alluxio/pull/13011

Comment (Xenorith) [reply docs/_config.yml:19]: wait, in the previous PR that updated versions, it was already called `edge`. i think its okay then

**Review by Xenorith — APPROVED** [review]

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**